en/docs/Learn/APIGateway/MessageMediation/change-the-default-mediation-flow-of-api-requests.md

```diff
@@ -59,329 +59,192 @@ Let's see how to create a custom sequence using the WSO2 API Manager Tooling Pl
-                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
-    | Sandbox endpoint    | [https://query.yahooapis.com/v1/public/yql](https://query.yahooapis.com/v1/public/yql?q=select%20*%20from%20weather.forecast%20where%20woeid%20in%20(select%20woeid%20from%20geo.places(1)%20where%20text%3D%22nome%2C%20ak%22)&format=json&env=store%3A%2F%2Fdatatables.org%2Falltableswithkeys)                                             |
-
-    ![](../../../assets/img/Learn/weatherapi-endpoint.png)
-
-4. Go to **Lifecycle** tab and **Publish** the API.
-    ![](../../../assets/img/Learn/lifecycle-publish.png)
+3. Go to RESOURCES tab and add below parameter to the the /order POST resource.
+    <table>
+    <thead>
+    <tr class="header">
+    <th>Parameters</th>
+    <th><br />
+    </th>
+    <th>Sample value</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr class="odd">
+    <td>Parameter Type</td>
+    <td><br />
+    </td>
+    <td>BODY</td>
+    </tr>
+    <tr class="even">
+    <td>Content-Type</td>
+    <td><br />
+    </td>
+    <td>application/json</td>
+    </tr>
+    </tbody>
+    </table>
+   ![](../../../assets/img/Learn/pizzaorder-add-parameter.png)
 
-5.  Download and install the WSO2 API Manager Tooling Plug-in by following one of the three possible methods described in Installing the API Manager Tooling Plug-In if you have not done so already. Start Eclipse by double clicking on the Eclipse application, which is  inside the downloaded folder.
+4. Download and install the WSO2 API Manager Tooling Plug-in by following one of the three possible methods described in Installing the API Manager Tooling Plug-In if you have not done so already. Start Eclipse by double clicking on the Eclipse application, which is  inside the downloaded folder.
 
-6.  Navigate to the **Window** menu, click **Perspective** , **Open Perspective,** and **Other** to open the Eclipse perspective selection window.
+5.  Navigate to the **Window** menu, click **Perspective** , **Open Perspective,** and **Other** to open the Eclipse perspective selection window.
```

Comment:
is this on Eclipse? If so shall we reword it so that it says this is on Eclipse?